Tasks/CopyFilesOverSSH/task.json

```diff
@@ -81,6 +81,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "failOnEmptySource",
+            "type": "boolean",
+            "label": "Fail when the Source folder is empty",
```

Comment:
In my opinion, "Source" should not be capitalized here.  It is capitalized only when used as the label of a field.